collector/pg_replication_slot.go

```diff
@@ -73,7 +91,9 @@ var (
 		    pg_current_wal_lsn() - '0/0'
 		END AS current_wal_lsn,
 		COALESCE(confirmed_flush_lsn, '0/0') - '0/0' AS confirmed_flush_lsn,
-		active
+		active,
+		safe_wal_size,
+		wal_status
```

Comment:
These were added in postgres 13, they are absent in 12 and below, so this will break scraping on Pg 12. See https://www.postgresql.org/docs/12/view-pg-replication-slots.html